### DISCUSSION TITLE
Positional border utilities

### DISCUSSION THREAD

**Discussion by Namchee** [tailwindlabs/tailwindcss#3901] [Ideas]
## Overview

By modern CSS features, it's possible to set border styles for distinct border positions either by using shorthand (e.g `border-left`) or applying them specifically one-by-one (e.g `border-left-color`).

I think it will be great if tailwind.css could have these individual properties in future releases.

**Comment by richardtallent** [comment]
Same. I have some situations like buttons attached to input elements where one border side needs to go away. Hopefully this is the kind of thing that JIT opens the door for, since otherwise it would require a ton of new classes.
